test/exec.js

```diff
@@ -35,7 +35,11 @@ test('config.fatal and unknown command', t => {
   shell.config.fatal = true;
   t.throws(() => {
     shell.exec('asdfasdf'); // could not find command
-  }, /exec: internal error/);
+    // the expected message depends on where tests are run:
+    //   /bin/sh says "foo: not found"
+    //   /bin/bash says "foo: command not found"
```

Comment:
We only need to support `/bin/sh` and `cmd.exe`, since these are the default shells. I think error messages are implementation specific, so let's just limit it to those 2 if we can get away with it (so long as mac, linux, and windows bots are happy).

Reply:
On my machine, at least, `/bin/bash` is the default shell.

Reply:
How about we regex for `asdfasdf` instead? I think that's good enough.